src/pages/programs/styles/Modal.css

```diff
@@ -0,0 +1,180 @@
+:root {
+    --text-link: #888b8f;
+    --darkest-color: #6f7172;
+    --even-midder: #dededf;
+    --button-color: #e8e9eb;
+    --mid-color: #d0d0d1;
+    --text-color: #6f7172;
+}
```

Comment:
see previous

Reply:
remember, React bundles everything together so styles are available everywhere in the project no matter what file they are in